engines/engine.cpp

```diff
@@ -310,37 +310,49 @@ void initGraphicsModes(const Graphics::ModeList &modes) {
 	g_system->initSizeHint(modes);
 }
 
-void initGraphics(int width, int height, const Graphics::PixelFormat *format) {
-
-	g_system->beginGFXTransaction();
+int initGraphicsAny(const Graphics::ModeWithFormatList &modes) {
+	int candidate = -1;
+	OSystem::TransactionError gfxError = OSystem::kTransactionSizeChangeFailed;
+	int last_width = 0, last_height = 0;
 
+	for (candidate = 0; candidate < (int) modes.size(); candidate++) {
```

Comment:
Please, do not insert space after type conversions, e.g. `(int)modes.size()`

Reply:
Done